src/lib/locals.js

```diff
@@ -1,18 +1,28 @@
 const {
   GTM_ANALYTICS_COOKIE_DOMAIN,
   GTM_ID,
+  GTM_ID_GA4,
   CDN_DOMAIN,
   CDN_PATH,
 } = require("./config");
 const { generateNonce } = require("./strings");
 
 module.exports = {
   setLocals: function (req, res, next) {
-    res.locals.gtmId = GTM_ID;
+    res.locals.uaContainerId = GTM_ID;
+    res.locals.ga4ContainerId = GTM_ID_GA4;
     res.locals.cspNonce = generateNonce();
     res.locals.analyticsCookieDomain = GTM_ANALYTICS_COOKIE_DOMAIN;
     res.locals.assetsCdnPath = CDN_PATH;
     res.locals.assetPath = CDN_DOMAIN + "/assets";
+
+    // Patch the status code setter to make it available in locals as well
+    const setStatusCode = res.status;
+    res.status = function (code) {
+      res.locals.statusCode = code;
+      return setStatusCode.call(res, code);
+    };
```

Comment:
It'd be nicer to read `res.statusCode` at the point of rendering, but I could work out a good way to do it